bolt/dashboard.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 var (
-	dashboardBucket         = []byte("dashboardsv2")
-	orgDashboardIndex       = []byte("orgsdashboardsv1")
-	dashboardCellViewBucket = []byte("dashboardcellviewsv1")
+	dashboardBucket         = []byte("dashboards/v2")
```

Comment:
This would invalidate all existing dashboards.  I'm ok with keeping the existing names here.

Reply:
@goller k

Reply:
should I also modify the kv buckets to reflect these?